webrender/src/platform/macos/font.rs

```diff
@@ -171,17 +171,16 @@ fn new_ct_font_with_variations(cg_font: &CGFont, size: f64, variations: &[FontVa
         if axes_ref.is_null() {
             return ct_font;
         }
-        let axes: CFArray = TCFType::wrap_under_create_rule(axes_ref);
+        let axes: CFArray<CFDictionary> = TCFType::wrap_under_create_rule(axes_ref);
         let mut vals: Vec<(CFString, CFNumber)> = Vec::with_capacity(variations.len() as usize);
-        for axis_ptr in axes.iter() {
-            let axis: CFDictionary = TCFType::wrap_under_get_rule(axis_ptr as CFDictionaryRef);
-            if !axis.instance_of::<CFDictionaryRef, CFDictionary>() {
+        for axis in axes.iter() {
+            if !axis.instance_of::<CFDictionary>() {
```

Comment:
I'm not sure if this check is needed. Well, if it's not needed now it was not needed before either, so maybe it covers some case I don't know about. But the documentation for `CTFontCopyVariationAxes` states that it returns an array of dictionaries, so not sure what item in the array might trigger this if branch.